test/resources/issue-101.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-function names (arg) {
+function names(arg) {
```

Comment:
These white spaces are intentional and are written with then everywhere. I thought that we had a linter rule for that as we have in dd-trace. The intention is to maintain javascript format as it is in dd-trace-js.

Reply:
I'm going to close this PR